### ISSUE TITLE
[Need Solution]: Even Fibonacci numbers

### ISSUE THREAD

**Issue by hmharshit** [hmharshit/hacktoberfest#4]
Question Name - Even Fibonacci numbers

**Comment by ektagarg** [comment]
Would like to take this up.[Golang]

**Comment by mcrd25** [comment]
Hi, is the expected output correct?
I'd just like to clear up what is expected. Are to return sum off all even number in fib sequence up to N sequence i.e if it were 10 `1, 2, 3, 5, 8, 13, 21, 34, 55, 89` where answer would be `44` or up or less than N in the sequence where answer would be `10`

**Comment by hmharshit** [comment]
@mcrd25 Yes, the expected output is correct! I've added explanation for the sample outputs.
![image](https://user-images.githubusercontent.com/21979074/66503050-1955b080-eae4-11e9-8f6f-c4a9a9c6c049.png)

**Comment by mcrd25** [comment]
@hmharshit 👋 ,

Thanks for clearing that up! Going to complete solution now using a bottom-up approach

**Comment by Pcat0** [comment]
I got this one in JS

**Comment by 22PoojaGaur** [comment]
Hi, I am doing this in python.

**Comment by AmanDeep9925** [comment]
I completed this in my favorite programming language i.e., C++
